core/templates/pages/topics-and-skills-dashboard-page/topics-list/topics-list.component.ts

```diff
@@ -156,7 +157,8 @@ export class TopicsListComponent {
     if (topic.getTotalUpcomingChaptersCount() > 1) {
       upcomingChapterNotificationsText += 'es';
     }
-    upcomingChapterNotificationsText += ' in the next 14 days';
+    upcomingChapterNotificationsText += ' in the next ' +
```

Comment:
Continuation lines should be indented to indicate scope:

```
    upcomingChapterNotificationsText += (
      ' in the next ' + ... +
      ...);
```

Reply:
Done